src/controllers/productController.js

```diff
@@ -0,0 +1,31 @@
+const productService = require("../services/productService");
+
+const productAllList = async (req, res) => {
+    const product = await productService.productAllList();
+    res.status(200).json(product);
+};
+
+const productCategoryList = async (req, res) => {
+    const categoryId = req.params.categoryStr;
+    const category = await productService.productCategoryList(categoryId);
+    res.status(200).json(category);
+};
+
+const productAromaList = async (req, res) => {
+    const aromasId  = req.params.aromasId;
+    const aromas = await productService.productAromaList(aromasId);
+    res.status(200).json(aromas);es.status(err.statusCode || 500).json({ message: err.message });
+};
+
+const productDetailPage = async (req, res) => {
```

Comment:
지금 모든 함수를 만들 때, 데이터 기준으로 만들어주세요. Page라는 용어는 프론트엔드를 생각하셔서 그런 것 같습니다.


```suggestion
const getProduct = async (req, res) => {
```